src/git-branch-delete.ts

```diff
@@ -30,11 +30,11 @@ const isGitRepository = (): boolean => {
 
 const getGitBranches = (): GitBranch[] => {
   const cmd =
-    "git for-each-ref --sort=committerdate refs/heads/ --format='%(HEAD) %(refname:short):::%(committerdate:relative):::%(committerdate:iso8601)'";
+    "git for-each-ref --sort=committerdate refs/heads/ --format=%(HEAD):%(refname:short):::%(committerdate:relative):::%(committerdate:iso8601)";
```

Comment:
Hi Kamil , thanks for your PR. 

I have tried your changes and I found that I have to put the ticks `'` back in. Otherwise the code breaks on MacOS. Are the ticks an issue on Windows?

Reply:
On Windows works either way, but in case format is enclosed with `'`, returns that tick `'` in output, so it needs to be calculated in later `filter` and `substring` functions.
Another way is to replace all `'` characters right after splitting by newlines. Branch name won't include `'` so it should not be conflicts there.